examples/fluids/qfunctions/newtonian.h

```diff
@@ -783,4 +785,41 @@ CEED_QFUNCTION(PressureOutflow_Jacobian_Prim)(void *ctx, CeedInt Q,
   return PressureOutflow_Jacobian(ctx, Q, in, out, StateFromY, StateFromY_fwd);
 }
 
+// *****************************************************************************
+// Harten Lax VanLeer (HLL) approximate Riemann solver.
+// Taking in two states (left, right) and returning Flux_HLL
+// *****************************************************************************
+CEED_QFUNCTION_HELPER StateConservative Harten_Lax_VanLeer_Flux(NewtonianIdealGasContext gas,
+    State left, State right, const CeedScalar normal[3]) {
```

Comment:
@jrwrigh Here is a Riemann solver you can call. The `normal` should be a unit vector pointing from left to right. Shall we try this on the top surface (at least) right away?